source/tutorials/monitor-minio-tenant.rst

```diff
@@ -62,6 +62,20 @@ The command returns a bearer token similar to the following:
      static_configs:
      - targets: ['HOSTNAME:9000']
 
+If using a self signed certificate, add the following to skip verification.
+
+.. code-block:: shell
+
+   scrape_config:
+   - job_name: minio-job
+     bearer_token: TOKEN
+     metrics_path: <PATH>
+     scheme: http
+     static_configs:
+     - targets: ['HOSTNAME:9000']
+     tls_config:
+       insecure_skip_verify: true
```

Comment:
it's possible that 

```suggestion
     tls_config:
       ca_file:  /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
```

This would be better instead?

Reply:
This assumes that auto-cert was enabled and used, right? There's also the option of the user having deployed with `externalCertSecret` so there are an unknown number of CAs which might be included.

Rather I'd like to just make a minor tweak to this stating that users can *either*:
- Configure prometheus to skip TLS verification, *or*
- Pass the CA used by the MinIO Tenant (given the hostname specified to `static_configs`.

@harshavardhana I think your suggestion only works if the Prometheus instance is also deployed in K8s and is using the K8s TLS API, so we can be generic ("Specify the path to the CA used to sign the MinIO Tenant. For MinIO Tenants deployed with autoCert enabled, Kubernetes cluster CA (Some link to docs here). If you deployed the Tenant using custom TLS certificates, specify the CA used to sign the certificate associated to the `static_configs[0].target` hostname.

Reply:
In this scenario we can expect that its deployed in k8s, since its a vsphere docs - I don't know if we need to talk about baremetal configs here.

Reply:
Ah fair point - I made a suggested change above to integrate your suggestion, as I'd prefer that over wholesale insecure TLS verification. Given that assumption, using `--insecure` on our end shouldn't be strictly required either...maybe the vsphere docs should include a "Connect to your Cluster" page that includes configuring `mc` and adding the Kubernetes CA to the system trust store. Would simplify things, I think.